api-specs.json

```diff
@@ -99,8 +99,8 @@
   "containerservice": {
     "dir": "azure-mgmt-containerservice",
     "source": "specification/containerservice/resource-manager/readme.md",
-    "package": "com.microsoft.azure.management.containerservice",
-    "args": "--payload-flattening-threshold=1 --tag=package-2019-08 --title=ContainerServiceManagementClient --description=\"Container Service Client\""
+    "package": "com.azure.management.containerservice",
+    "args": "--payload-flattening-threshold=1 --tag=package-2019-08 --title=ContainerServiceManagementClient --description=\"Container Service Client\" --track1-naming=true"
```

Comment:
what's track1-naming mean?

Reply:
The track1-naming is used to generate code with naming as track1, "with***" instead of "set***". cc @weidongxu-microsoft

Reply:
I probably going to put it to gulp.js, so this is enabled globally.

Reply:
Should keep track-1 as default. And add 'azure-core-style' as alternative.